Tell us about yourself update

courtneyjacinic opened the pull request:
Addressed: reliable transportation, humane organization, none of the above experience, getting medication (on this one, I changed the language slightly and added in some punctuation that may clarify. The way it is currently written is honestly the clearest.)

courtneyjacinic committed:
Tell us about yourself update

reliable transportation, humane organization, none of the above experience, getting medication (on this one, I changed the language slightly and added in some punctuation that may clarify. The way it is currently written is honestly the clearest)

juliazbyron commented on _pages/Yourself_info.html:
```diff
@@ -39,9 +39,9 @@ <h2 id="page_header">Tell us about yourself</h2>
+        <legend>Do you have reliable transportation, such as your own car or access to a car whenever you need it, to bring your foster animal to the Austin Animal Center for free foster animal veterinary care or in the event of an emergency?</legend>
         <small id="transportationHelpBlock" class="form-text text-muted">
-            We provide all veterinary care at no cost to you.
+            Note that city buses and taxis usually only allow service animals on board, and ride share drivers do not always allow animals and are not considered reliable.
```
I'd remove the part about not ride share not being considered reliable because AAC indicated they would if they took the pets. We don't need to make judgment calls on what's reliable, just give the guiding info.

courtneyjacinic committed:
Home permission and children visiting questions

-Do you have permission question added. Needs js change to be made conditional to answering "No" to above question of "Do you own your home?"
-Visiting often children added to last two questions.

courtneyjacinic committed:
Home permission alert change

Minor changes to the alert to be a little more friendly. Kept the word home instead of saying "house or condo" because it is friendlier and people can get confused with house/condo/apartment/bungalow/flat/townhouse. It's a slippery slope to specify.

courtneyjacinic committed:
Often visiting animals

courtneyjacinic committed:
Potential Adopters

Clarified language in the question to define "potential adopters" and removing the possessive "your" from foster animal.

courtneyjacinic committed:
Combine final free response questions

courtneyjacinic committed:
reliable transportation

in light of learning that there are pet specific ride share companies in Austin, I edited this helper text again.

courtneyjacinic committed:
Added pet transport company mention

you can decide whether or not to keep it in.

juliazbyron commented on _pages/Home_info.html:
```diff
@@ -23,19 +23,36 @@ <h2 id="page_header">Tell us about your home</h2>
             </div>
         </fieldset>
     </div>
-    <div>
```
I think we shouldn't be deleting this "Add additional info" option, right?

juliazbyron commented on _pages/Yourself_info.html:
```diff
@@ -39,9 +39,9 @@ <h2 id="page_header">Tell us about yourself</h2>
+        <legend>Do you have reliable transportation, such as your own car or access to a car whenever you need it, to bring your foster animal to the Austin Animal Center for free foster animal veterinary care or in the event of an emergency?</legend>
         <small id="transportationHelpBlock" class="form-text text-muted">
-            We provide all veterinary care at no cost to you.
+            Note that city buses only allow service animals on board, and ride share drivers do not always allow animals. There may be pet transportation companies in your area.
```
Should we say "_taxi and_ ride share drivers"?

juliazbyron commented on _pages/Yourself_info.html:
```diff
@@ -89,7 +89,7 @@ <h2 id="page_header">Tell us about yourself</h2>
             <i class="fas fa-exclamation-triangle fa-2x fa-fw col-md-1"></i>
             <div class="col-md-11 alert-body">
-              <p>You may continue with the application, but if you cannot retrieve medications you should consider only short term foster periods of two weeks or less.</p>
+              <p>You may continue with the application, but if you cannot go get medications, you should consider only doing short term foster periods of two weeks or less.</p>
```
Putting this here bc I can't see how to put it on line 73 - should we change 'preventative' to 'prevention' or something else simpler?

juliazbyron commented on _pages/Yourself_info.html:
```diff
@@ -149,6 +149,12 @@ <h2 id="page_header">Tell us about yourself</h2>
             Behavioral issues, basic obedience training, or socialization
```
Change 'behavioral' to 'behavior'?

juliazbyron reviewed (changes requested):
Overall great, just a couple tiny things.

juliazbyron commented:
@courtneyjacinic I need to use the new copy so I merged the request, but you can still make the edits.
